### PR TITLE
add axzhir.md

### PR DESCRIPTION
**Pull request by axzhir**
https://github.com/axzihr/axzhir.github.io/blob/add-axzhir-profile/axzhir.md

https://raw.githack.com/axzhir/axzhir.github.io/add-axzhir-profile/index.html#!pages/mi/profiles/axzhir.md

### CODE REVIEW

File: axzhir.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+    1. Python
+    2. Java
+    3. Kotlin
+    4. Swift
+
+_**Social Links:**_
+
+    1. LinkedIn -> https://www.linkedin.com/in/fttahir/
+    2. GitHub -> https://github.com/axzihr
```
Comment (xyb994) [inline axzhir.md:21]: @axzhir It looks like your github username got misspelled.

**Review by xyb994 — CHANGES_REQUESTED** [review]
Could you also edit the pull request description and provide the raw.githack preview link to your profile as laid out in the pull  request template / step 1?

**Review by dogi — CHANGES_REQUESTED** [review]
please use some more complex markdown elements
- link
- picture
- table

**Review by xyb994 — CHANGES_REQUESTED** [review]
The file appear to be located directly in the repository's root folder while we meant to ask you to add it to `pages/mi/profiles/` folder. Could you move it with the proper directory at https://docs.github.com/en/repositories/working-with-files/managing-files/moving-a-file-to-a-new-location ?

**Review by dogi — APPROVED** [review]

**Comment by Okuro3499** [comment]
@axzhir I can noww see your profile could you check https://open-learning-exchange.github.io/#!pages/mi/mi-10-steps.md#Step_8_-_Create_Issues_and_Pull_Requests to update your pr title to include the issue number

**Review by Okuro3499 — CHANGES_REQUESTED** [review]
@axzhir I can now see your profile could you check https://open-learning-exchange.github.io/#!pages/mi/mi-10-steps.md#Step_8_-_Create_Issues_and_Pull_Requests to update your pr title to include the issue number

**Review by xyb994 — APPROVED** [review]

**Pull request merged**